tests/greenheart/test_greenheart_system.py

```diff
@@ -81,7 +81,7 @@ def test_simulation_wind(subtests):
 
     with subtests.test("lcoh"):
         assert lcoh == approx(
-            7.056407257515502
+            7.248875250895419
```

Comment:
I'm inclined to use the fixed sizing for these tests and just use the variable sizing in the GreenSteel tests so we avoid changing test values and keep some tests for the fixed storage size approach

Reply:
These value changes are coming from using `electrolyzer_physics_results['H2_Results']['Time Until Replacement [hrs]']` rather than the `time_between_replacement` for the electrolyzer schedule, not changing hydrogen sizing.